### PR TITLE
Blog foodsearchinc

### PR DESCRIPTION
**Pull request by johnnewton-hub**
Hi Aaron,

Here's hopin' this one goes down smoothly.  Thanks in advance.

jn

### CODE REVIEW

File: _posts/2020-04-16-foodsearchinc.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+author: John
+title: foodsearchinc.com connects customers and the food and restaurant industry
```
Comment (ajyong) [inline _posts/2020-04-16-foodsearchinc.md]: - Needs title-case
- Needs shortening

 Consider:
> FoodSearchInc Connects Customers & Restaurants During the Pandemic

File: _posts/2020-04-16-foodsearchinc.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+---
+
+A co-winner of our week four competition, [foodsearchinc.com](https://foodsearchinc.com/), hopes to connect people to comprehensive information about the food they desire, and how to easily locate it.
```
Comment (ajyong) [inline _posts/2020-04-16-foodsearchinc.md:6]: Let's respect their branding style and be consistent with their name.  From what I can tell, it is `FoodSearchInc`.  I could be wrong though.  There are resources for this somewhere on our Drive.

File: _posts/2020-04-16-foodsearchinc.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+author: John
+title: foodsearchinc.com connects customers and the food and restaurant industry
+---
+
+A co-winner of our week four competition, [foodsearchinc.com](https://foodsearchinc.com/), hopes to connect people to comprehensive information about the food they desire, and how to easily locate it. 
+
+
```
Comment (ajyong) [inline _posts/2020-04-16-foodsearchinc.md]: Remove blank line

File: _posts/2020-04-16-foodsearchinc.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+<img src="/images/blog/foodsearchinc.png" alt="Get Cleard" style="max-width: 100%;">
+
+Many thanks to [Vinson Luong]( https://www.linkedin.com/in/vinson-luong-b4682662/) for sharing the story of foodsearchinc.com with us.  The responses are his, lightly edited for length and style.
```
Comment (ajyong) [inline _posts/2020-04-16-foodsearchinc.md:10]: Stylize the product name

File: _posts/2020-04-16-foodsearchinc.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  
+**How did your team come together?**
+
+Our company, Food Search Inc., was created to increase the quality of information, and facilitate access to the food we eat, by creating a database of products and dishes with information such as:  ingredients, allergens, and link this to information about where to find it. In January 2020, [Tyler](https://www.linkedin.com/in/tylerfindlay/) and I started our first project [searchglutenfree.com](https://searchglutenfree.com/), which helps individuals with celiac disease or gluten intolerance, find restaurants and dishes much easier. Due to the pandemic, we decided to repurpose the code, and created foodsearchinc.com to address problems faced by the food industry. Our team was made possible through networking opportunities facilitated by [Startup Edmonton](https://www.startupedmonton.com/ ). A few chilled beers and meetings later, we teamed up to change the way people find information about the food they eat.
```
Comment (ajyong) [inline _posts/2020-04-16-foodsearchinc.md:24]: Is "Food Search Inc." how they want to be branded?  Then if so, we should use this more often than `foodsearchinc.com`.  I only use URLs when I want to make it abundantly clear that it's a link; we don't need to be that blunt all the time.

File: _posts/2020-04-16-foodsearchinc.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+**Where is your team based?**
+
+Tyler Findlay and I are from Edmonton.
+
+**Can you describe the skills you bring to the project?**
+
+Tyler, the technical co-founder is a full-stack software developer and scrum master. I am a non-technical co-founder, and manage the operational and business development side.
```
Comment (ajyong) [inline _posts/2020-04-16-foodsearchinc.md]: >scrum

Is capitalized as Scrum.

File: _posts/2020-04-16-foodsearchinc.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+Rewarding, and solving a problem in real time while speaking with over 400 restaurant owners has been fulfilling. 
+
+**What tools, made available by the  hackathon, are you using?**
+
+[*Slack*](https://app.slack.com/client/TV5MXDY8Z/D012J7RHR7A), and [*FindCollabs*](https://findcollabs.com/hackathon/codevid-19-isp21fkqtjupchx7kjed)
+
+**Is there anything else you’d like to say?**
+
+Thank you to the team that put Codevid-19 together! It’s such an amazing initiative for creating projects to help out during these uncertain times.
```
Comment (ajyong) [inline _posts/2020-04-16-foodsearchinc.md]: CODEVID-19

File: _posts/2020-04-16-foodsearchinc.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+Thank you from Codevid-19 for your fine effort, and best of luck in the future.
```
Comment (ajyong) [inline _posts/2020-04-16-foodsearchinc.md]: CODEVID-19

**Review by ajyong — APPROVED** [review]

**Comment by ajyong** [comment]
Oh I uhh... meant to hit "Request Changes" and misclicked.  See feedback.

File: _posts/2020-04-16-foodsearchinc.md (hunk 1 above)
Comment (johnnewton-hub) [reply _posts/2020-04-16-foodsearchinc.md]: I did check into their naming/branding , on their contest entry info, website, and images.  It was pretty much everywhere as I wrote it, I hate the way it looks too, but I deferred to their sense of style. It also makes the title look stupid as you note.  So consistency v. grammar, who wins?

Comment (johnny-newton) [reply _posts/2020-04-16-foodsearchinc.md:3]: see next comment branding and the title format are related

File: _posts/2020-04-16-foodsearchinc.md (hunk 2 above)
Comment (johnnewton-hub) [reply _posts/2020-04-16-foodsearchinc.md:6]: see above, but we could make an executive decision that suits us too...not sure what's best here.

File: _posts/2020-04-16-foodsearchinc.md (hunk 5 above)
Comment (johnnewton-hub) [reply _posts/2020-04-16-foodsearchinc.md:24]: "Food Search Inc." is the company name, not the project name.

File: _posts/2020-04-16-foodsearchinc.md (hunk 1 above)
Comment (ajyong) [reply _posts/2020-04-16-foodsearchinc.md]: If that's how their submission was entered, let's go with that then.

File: _posts/2020-04-16-foodsearchinc.md (hunk 2 above)
Comment (ajyong) [reply _posts/2020-04-16-foodsearchinc.md:6]: Based on prior comment, this is their style, so we can opt to keep it that way.  It just didn't initially strike me as deliberate.

**Comment by johnnewton-hub** [comment]
I’ve just DM’d the project leader-no surprise it’s unclear-you should have seen the rest of it…major edit this am.

**Comment by johnnewton-hub** [comment]
sadly, they want foodsearchinc.com <http://foodsearchinc.com/>,  their call

File: _posts/2020-04-16-foodsearchinc.md (hunk 4 above)
Comment (ajyong) [reply _posts/2020-04-16-foodsearchinc.md:10]: Nevermind!  That is their stylized name.

**Comment by ajyong** [comment]
All good, thanks for getting clarification.  They have full say on how it looks, I just wanted to be correct.  And thanks for doing a heavy edit of their original copy!

File: _posts/2020-04-16-foodsearchinc.md (hunk 3 above)
Comment (johnnewton-hub) [reply _posts/2020-04-16-foodsearchinc.md]: done

Comment (ajyong) [reply _posts/2020-04-16-foodsearchinc.md]: I think the wrong blank line got removed, this one's still around... but now there isn't one between the front matter (the author/title section) and the excerpt paragraph.

File: _posts/2020-04-16-foodsearchinc.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+---
+author: John
+title: foodsearchinc.com connects customers & restaurants during pandemic
+---
+A co-winner of our week four competition, [foodsearchinc.com](https://foodsearchinc.com/), hopes to connect people to comprehensive information about the food they desire, and how to easily locate it.
```
Comment (ajyong) [inline _posts/2020-04-16-foodsearchinc.md]: Add blank line just above here

File: _posts/2020-04-16-foodsearchinc.md (hunk 7 above)
Comment (ajyong) [reply _posts/2020-04-16-foodsearchinc.md]: You're probably already on it, but just a reminder

File: _posts/2020-04-16-foodsearchinc.md (hunk 8 above)
Comment (ajyong) [reply _posts/2020-04-16-foodsearchinc.md]: Bump

**Review by ajyong — CHANGES_REQUESTED** [review]
Alright, the last 4 that my tired eyes have spotted.  This is coming along very nicely!

File: _posts/2020-04-16-foodsearchinc.md (hunk 7 above)
Comment (ajyong) [reply _posts/2020-04-16-foodsearchinc.md]: Weird, I do not see this change, and I've manual refreshed a few times.  I also checked the commits and not one has a diff showing the necessary correction.

**Comment by johnnewton-hub** [comment]
give me a sec…dude you are soooo fasssst.

**Comment by ajyong** [comment]
Don't mean to rush you, I noticed some conversations were resolved before
changes were applied. I'm pretty new to this PR system myself, but my
assumption was that resolved = committed

File: _posts/2020-04-16-foodsearchinc.md (hunk 9 above)
Comment (johnnewton-hub) [reply _posts/2020-04-16-foodsearchinc.md]: sorry where is here-this thing needs a highlighter of some sort...I'll take a quick look and see if it's obvious in the code.

Comment (johnnewton-hub) [reply _posts/2020-04-16-foodsearchinc.md]: Generally is the highlighted (green) code showing me what I have, or what you'd like?

**Comment by ajyong** [comment]
Neither, it means code that was added or changed.

I believe the blank line needs to be added between line 4 and 5.

File: _posts/2020-04-16-foodsearchinc.md (hunk 7 above)
Comment (johnnewton-hub) [reply _posts/2020-04-16-foodsearchinc.md]: I may have resolved it accidentally.  How do I restore the comment?

**Comment by ajyong** [comment]
I did it for you, but you can un-collapse a resolved conversation and press
"Unresolve Conversation" next time.

File: _posts/2020-04-16-foodsearchinc.md (hunk 8 above)
Comment (johnnewton-hub) [reply _posts/2020-04-16-foodsearchinc.md]: qu'est-ce que sais "Bump" If that's slang, don't forget that I'm old.

**Comment by ajyong** [comment]
Just a way to surface a prior comment just in case it gets overlooked

**Comment by johnnewton-hub** [comment]
absolutely-I messed that up, but am on it now…probs just navigating to where the correction are made—others too, but that one as well-I think I’m closer, but a bit confused occasionally by the descriptions of what is to be changed, I’m likely missing something...

**Comment by johnnewton-hub** [comment]
thx I’ll check

jn

**Comment by johnnewton-hub** [comment]
thx

**Comment by johnnewton-hub** [comment]
like "bump it up into view” thanks -old dogs-new tricks

**Review by johnnewton-hub — COMMENTED** [review]
I think this is it...

File: _posts/2020-04-16-foodsearchinc.md (hunk 7 above)
Comment (ajyong) [reply _posts/2020-04-16-foodsearchinc.md]: This isn't done, please double check the Changes tab for any outstanding comments before asking for a review.

File: _posts/2020-04-16-foodsearchinc.md (hunk 8 above)
Comment (ajyong) [reply _posts/2020-04-16-foodsearchinc.md]: Same here, see original comment

**Review by ajyong — CHANGES_REQUESTED** [review]

**Comment by johnnewton-hub** [comment]
I'm pulling my hair out trying to find where this original comment is for this and the other fix, so I can sort this. I appear to be in the right place-but wtf?  

j

**Comment by ajyong** [comment]
I don't know, because I can't be sure what's happening on your end.  I'll just make the 2 changes remaining and approve it.

**Review by ajyong — APPROVED** [review]
LGTM, feel free to merge

**Review by johnnewton-hub — COMMENTED** [review]
Thanks Aaron-I see what you've done.  I've been at the tutorials.

**Comment by johnnewton-hub** [comment]
I was just about to ask!  I was in there messing around while you fixed it!

**Pull request merged**